tests/contrib/kombu/test.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+import kombu
+from nose.tools import eq_
+
+from ddtrace import Pin
+from ddtrace.contrib.kombu import get_traced_kombu
+from ddtrace.contrib.kombu.patch import patch, unpatch
+from ..config import RABBITMQ_CONFIG
+from ...test_tracer import get_dummy_tracer
+
+
+def test_kombu_legacy():
+    results = []
+
+    def process_message(body, message):
+        results.append(body)
+        message.ack()
+
+    # ensure the old interface isn't broken, but doesn't trace
+    tracer = get_dummy_tracer()
+    TracedKombu = get_traced_kombu(tracer, "foo")
+    conn = TracedKombu("amqp://guest:guest@127.0.0.1:%s//" % (RABBITMQ_CONFIG['port'], ))
```

Comment:
Let's stick to the library convention of using `"".format()` over string placeholders.